ide/schema2beans/src/org/netbeans/modules/schema2beansdev/gen/JavaUtil.java

```diff
@@ -874,10 +874,16 @@ public static int getOptimialHashMapSize(Object[] keys, int maxSize) {
                 }
                 return table.length;
             }
-        } catch (java.lang.IllegalAccessException e) {
-            return defaultAnswer;
-        } catch (java.lang.NoSuchFieldException e) {
+        } catch (IllegalAccessException | NoSuchFieldException e) {
             return defaultAnswer;
+        } catch (RuntimeException ex) {
+            // todo: remove this workaround post JDK 9+ migration
+            // modules on modern JDKs should prefere immutable collections anyway, e.g Map.of();
```

Comment:
There is a way to use `Map.of`. Described in my [Run on JDK8, use JDK11 APIs post](http://wiki.apidesign.org/wiki/AlternativeImplementation) - if you believe this cases deserves that design pattern (I don't), feel free to do implement it.

Reply:
please no - things are complex enough already lets try to remove workarounds instead of adding more :)